graylog2-web-interface/src/components/events/events/ColumnRenderers.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import * as React from 'react';
+import { useMemo } from 'react';
+import type Immutable from 'immutable';
+import styled from 'styled-components';
+import isEmpty from 'lodash/isEmpty';
+
+import { isPermitted } from 'util/PermissionsMixin';
+import type { ColumnRenderers } from 'components/common/EntityDataTable';
+import EventTypeLabel from 'components/events/events/EventTypeLabel';
+import { Link } from 'components/common/router';
+import Routes from 'routing/Routes';
+import type { Event, EventsAdditionalData } from 'components/events/events/types';
+import PriorityName from 'components/events/events/PriorityName';
+import usePluginEntities from 'hooks/usePluginEntities';
+import EventFields from 'components/events/events/EventFields';
+import { MarkdownPreview } from 'components/common/MarkdownEditor';
+import useExpandedSections from 'components/common/EntityDataTable/hooks/useExpandedSections';
+import { Timestamp } from 'components/common';
+import useCurrentUser from 'hooks/useCurrentUser';
+
+const EventDefinitionRenderer = ({ eventDefinitionId, permissions, meta }: { eventDefinitionId: string, permissions: Immutable.List<string>, meta: EventsAdditionalData }) => {
+  const { context: eventsContext } = meta;
+  const eventDefinitionContext = eventsContext?.event_definitions?.[eventDefinitionId];
+
+  if (!eventDefinitionContext) {
+    return <em>{eventDefinitionId}</em>;
+  }
+
+  if (isPermitted(permissions, `eventdefinitions:edit:${eventDefinitionContext.id}`)) {
+    return (
+      <Link to={Routes.ALERTS.DEFINITIONS.edit(eventDefinitionContext.id)}>
+        {eventDefinitionContext.title}
+      </Link>
+    );
+  }
+
+  return <>{eventDefinitionContext.title}</>;
+};
```

Comment:
If I remember correctly we also discussed replacing the `permissions` props with using `useCurrentUser` here.